docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services:
+  ftpserver:
+    build: .
+    ports:
+      - "32021:32021"
```

Comment:
This probably should match vsftpd.conf. It's not actually being used by `ftpclient` (since they share a network) but just for consistency (or remove)